src/cloud/project/routes.md

```diff
@@ -103,12 +128,12 @@ Property         | Description
 
 ## Simple routes
 
-The following sample routes the apex domain and the `www` subdomain to the `frontend` application. This route does not redirect HTTPS requests:
+The following sample routes the apex domain and the `www` subdomain to the `mymagento` application. This route does not redirect HTTPS requests:
```

Comment:
I am a little confused by the sentence: "This route does not redirect HTTPS requests" -- Since HTTPS is the standard now, why would we want to provide an example that doesn't redirect HTTPS requests. 

```suggestion
In the following examples, the route configuration routes the apex domain and the `www` subdomain to the `mymagento` application. This route does not redirect HTTPS requests.

**Example 1:**

```